cfme/containers/image.py

```diff
@@ -129,21 +128,8 @@ def compliant(self):
 @attr.s
 class ImageCollection(BaseCollection):
     """Collection object for :py:class:`Image`."""
-    ENTITY = Image
-
-    def all(self):
```

Comment:
Add this back in, will modify in future PRs

Reply:
Done